app/models/manageiq/providers/nuage/network_manager/network_router.rb

```diff
@@ -1,3 +1,4 @@
 class ManageIQ::Providers::Nuage::NetworkManager::NetworkRouter < ::NetworkRouter
-  has_many :floating_ips, :dependent => :destroy
+  # TODO(miha-plesko): uncomment when https://github.com/ManageIQ/manageiq-schema/pull/217 is merged
+  # has_many :floating_ips, :dependent => :destroy
```

Comment:
the best might be to define:

```ruby
def floating_ips
  FloatingIp.none
end
```

so you can have it used in UI or other places, but it will just return blank result until properly implemented

Reply:
Makes sense, done.